digitalocean/table_digitalocean_database.go

```diff
@@ -118,6 +119,63 @@ func getDatabase(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData
 	return *result, nil
 }
 
+func getUsers(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	conn, err := connect(ctx, d)
+	if err != nil {
+		plugin.Logger(ctx).Error("digitalocean_database.getUsers", "connection_error", err)
+		return nil, err
+	}
+	id := h.Item.(godo.Database).ID
+	users, resp, err := conn.Databases.ListUsers(ctx, id, nil)
+	if err != nil {
+		if strings.Contains(err.Error(), ": 404") {
+			plugin.Logger(ctx).Warn("digitalocean_database.getUsers", "not_found_error", err, "id", id, "resp", resp)
+			return nil, nil
+		}
+		plugin.Logger(ctx).Error("digitalocean_database.getUsers", "query_error", err, "id", id, "resp", resp)
+		return nil, err
+	}
+	return users, nil
+}
+
+func getDbs(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
```

Comment:
```suggestion
func getDatabaseNames(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
```
Also change the same in log statement.